### PR TITLE
Checkbox for medicare premium help

### PR DESCRIPTION
**Pull request by disambiguator**
* if person has Medicare, then we should automatically check "Yes" on the
   1426 field, "Do you need help paying Medicare premiums?"

[Finishes #152727191]

https://www.pivotaltracker.com/story/show/152727191

Signed-off-by: Ben Golder <bgolder@codeforamerica.org>

### CODE REVIEW

File: spec/models/medicaid_application_attributes_spec.rb [hunk 1]
```diff
@@ -21,7 +21,29 @@
 
     describe "insurance types" do
       it_should_behave_like "insurance type", "medicaid", "Medicaid"
-      it_should_behave_like "insurance type", "medicare", "Medicare"
+
+      context "applicant is on medicare" do
```
Comment (jessieay) [inline spec/models/medicaid_application_attributes_spec.rb]: shoud we add "and needs help paying medical bills from the past 3 months" to this  context?

File: spec/support/shared_examples/insurance_type.rb [hunk 2]
```diff
@@ -3,20 +3,20 @@
 RSpec.shared_examples_for "insurance type" do |insurance_slug, insurance_name|
   context "one household member with #{insurance_name}" do
     it "sets insured_#{insurance_slug}_yes to 'Yes' and lists their name" do
-      member_with_medicaid = build(
+      member_with_insurance = build(
```
Comment (jessieay) [inline spec/support/shared_examples/insurance_type.rb]: 💯

File: spec/models/medicaid_application_attributes_spec.rb (hunk 1 above)
Comment (disambiguator) [reply spec/models/medicaid_application_attributes_spec.rb]: That feels unrelated. This is the box we're checking.

pasted image at 2017_11_21_06_03 pm

Comment (jessieay) [reply spec/models/medicaid_application_attributes_spec.rb]: gotcha, I interpreted this feature to be related to that question but it sounds like it isn't

Comment (disambiguator) [reply spec/models/medicaid_application_attributes_spec.rb]: Yee

**Pull request merged**